guice-property/src/test/java/com/maxifier/guice/property/FilesPropertyProviderTest.java

```diff
@@ -0,0 +1,68 @@
+package com.maxifier.guice.property;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.ImmutableMap;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Properties;
+
+public class FilesPropertyProviderTest {
```

Comment:
Strange name, there is no such class `FilesPropertyProvider`

Reply:
Fixed